.env.prod

```diff
@@ -22,6 +22,17 @@ REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x2da719db753dfa10a62e140f436e1d67f2ddb0
 REACT_APP_ETHEREUM_ERC20_NAME="CERE Network"
 REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
 
+### Polygon
+REACT_APP_POLYGON_CHAIN_ID="137"
+REACT_APP_POLYGON_NETWORK_ID="137"
+REACT_APP_POLYGON_NETWORK_NAME="Polygon Mainnet"
+REACT_APP_POLYGON_BRIDGE_ADDRESS="update mainnet bridge smart contract address"
+REACT_APP_POLYGON_ERC20_HANDLER_ADDRESS="update mainnet ERC20 handler smart contract address"
+REACT_APP_POLYGON_RPC_URL=""

```

Comment:
Do we need this parameter at all?

Reply:
No, We can hard code it. Here is the ticket for [it](https://cerenetwork.atlassian.net/browse/CBI-1422).